specs/PrintToPdf.md

```diff
@@ -0,0 +1,294 @@
+
+  /// The page height in inches. The default height is 11 inches.
+  [propget] HRESULT PageHeight([out, retval] double* pageHeight);
+
+  /// Sets the `PageHeight` property.
+  [propput] HRESULT PageHeight([in] double pageHeight);
+
+  /// The top margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginTop([out, retval] double* marginTop);
+
+  /// Sets the `MarginTop` property.
+  [propput] HRESULT MarginTop([in] double marginTop);
+
+  /// The bottom margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginBottom([out, retval] double* marginBottom);
+
+  /// Sets the `MarginBottom` property.
+  [propput] HRESULT MarginBottom([in] double marginBottom);
+
+  /// The left margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginLeft([out, retval] double* marginLeft);
+
+  /// Sets the `MarginLeft` property.
+  [propput] HRESULT MarginLeft([in] double marginLeft);
+
+  /// The right margin in inches. The default is 1 cm, or ~0.4 inches.
+  [propget] HRESULT MarginRight([out, retval] double* marginRight);
+
+  /// Set the `MarginRight` property.
+  [propput] HRESULT MarginRight([in] double marginRight);
+
+  /// `TRUE` if backgrounds should be printed. The default value is `FALSE`.
+  [propget] HRESULT ShouldPrintBackgrounds(
+      [out, retval] BOOL* shouldPrintBackgrounds);
+
+  /// Set the `ShouldPrintBackgrounds` property.
+  [propput] HRESULT ShouldPrintBackgrounds([in] BOOL shouldPrintBackgrounds);
+
+  /// `TRUE` if only the selection should be printed. The default value is `FALSE`.
+  [propget] HRESULT ShouldPrintSelectionOnly(
+      [out, retval] BOOL* shouldPrintSelectionOnly);
+
+  /// Set the `ShouldPrintSelectionOnly` property.
+  [propput] HRESULT ShouldPrintSelectionOnly(
+      [in] BOOL shouldPrintSelectionOnly);
+
+  /// `TRUE` if header and footer should be printed. The default value is `FALSE`.
+  [propget] HRESULT ShouldPrintHeaderFooter(
+      [out, retval] BOOL* shouldPrintHeaderFooter);
+
+  /// Set the `ShouldPrintHeaderFooter` property.
+  [propput] HRESULT ShouldPrintHeaderFooter([in] BOOL shouldPrintHeaderFooter);
+
+  /// The title in the header if `ShouldPrintHeaderFooter` is `TRUE`.
+  [propget] HRESULT HeaderTitle([out, retval] LPWSTR* headerTitle);
+
+  /// Set the `HeaderTitle` property.
+  [propput] HRESULT HeaderTitle([in] LPCWSTR headerTitle);
+
+  /// The URI in the footer if `ShouldPrintHeaderFooter` is `TRUE`.
+  [propget] HRESULT FooterUri([out, retval] LPWSTR* footerUri);
+
+  /// Set the `FooterUri` property.
+  [propput] HRESULT FooterUri([in] LPCWSTR footerUri);
+}
+
+[uuid(7dee69a9-c2cc-422b-9f86-e6ed6551ce95), object, pointer_default(unique)]
+interface ICoreWebView2Environment5 : IUnknown
+{
+    /// Creates the `ICoreWebView2PrintSettings` used by the `PrintToPdf`
+    /// method with the following default values:
+    /// - pageOrientation: COREWEBVIEW2_PRINT_ORIENTATION_PORTRAIT
```

Comment:
I'm thinking you don't need to restate all the defaults here since you document them per property in the print settings docs already